apps/frontend/src/components/chat/ChatPane.tsx

```
@@ -40,18 +72,32 @@ const ChatPane: React.FC = () => {
+            <div ref={messagesEndRef} />
+          </>
         )}
       </div>
 
+      {/* Scroll to bottom button */}
+      {showScrollButton && currentThread.messages.length > 0 && (
+        <button
+          onClick={scrollToBottom}
+          className="absolute bottom-4 right-4 bg-blue-600 hover:bg-blue-700 text-white rounded-full p-2 shadow-lg transition-all duration-200 z-10"
+          title="Scroll to bottom"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)